Packs/FeedInfobloxThreatIntelligence/Integrations/FeedInfobloxThreatIntelligence/README.md

```diff
@@ -0,0 +1,259 @@
+            },
+            {
+                "id": "00000000-0000-0000-0000-000000000004",
+                "type": "HASH",
+                "hash": "000000000000000000000000000000000000000000000000000000000000001",
+                "hash_type": "SHA256",
+                "profile": "IID",
+                "property": "MalwareC2_Azorult",
+                "class": "MalwareC2",
+                "threat_level": 100,
+                "expiration": "2025-08-25T20:00:34.12Z",
+                "detected": "2024-08-25T20:00:34.12Z",
+                "received": "2024-08-25T20:01:35.75Z",
+                "imported": "2024-08-25T20:01:35.75Z",
+                "up": "true",
+                "confidence": 100,
+                "batch_id": "00000000-0000-0000-0000-000000000005",
+                "extended": {
+                    "cyberint_guid": "00000000000000000000000000000005",
+                    "notes": "The file uses Living off the Land (LotL) methods, a malicious technique that involves abusing pre-built software on the victim's machine to execute attacks. This binary matches the threat signature",
+                    "sample_sha256": "000000000000000000000000000000000000000000000000000000000000001"
+                }
+            },
+            {
+                "id": "00000000-0000-0000-0000-000000000005",
+                "type": "URL",
+                "ip": "0.0.0.0",
+                "url": "http://0.0.0.0/example",
+                "profile": "IID",
+                "property": "MalwareC2_Generic",
+                "class": "MalwareC2",
+                "threat_level": 100,
+                "expiration": "2025-10-01T10:32:58.891Z",
+                "detected": "2025-06-03T10:32:58.891Z",
+                "received": "2025-06-03T10:36:37.283Z",
+                "imported": "2025-06-03T10:36:37.283Z",
+                "up": "true",
+                "confidence": 100,
+                "batch_id": "00000000-0000-0000-0000-000000000005",
+                "extended": {
+                    "attack_chain": "ACIS",
+                    "cyberint_guid": "00000000000000000000000000000005",
+                    "notes": "Exfiltration target of Cuckoo Stealer for MacOS (SHA256: 00000000000000000000000000000005)",
+                    "protocol": "http",
+                    "references": "https://www.example.com/gui/file/00000000000000000000000000000005"
+                }
+            }
+        ]
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Infoblox TIDE Indicators: Found 5 indicators between 2023-01-01T00:00:00.000Z and 2024-12-31T23:59:59.999Z
+>
+>|Type|Value|Threat Class|Confidence|Threat Level|Expiration|Property|Profile|
+>|---|---|---|---|---|---|---|---|
+>| Email | test@example.com | APT | 100 | 80 | 2043-01-06T00:41:57.421Z | APT_testC2 | IID |
+>| IP | 0.0.0.0 | APT | 100 | 100 | 2042-11-01T09:29:18.721Z | APT_testC2 | IID |
+>| Domain | test.net | MalwareC2 | 100 | 100 | 2026-04-15T23:54:58.665Z | MalwareC2_testRAT | IID |
```

Comment:
The `Type` column in the 'Human Readable Output' table uses values that differ from the `type` field in the context. For example, the table shows 'Domain' and 'File', while the context example shows 'HOST' and 'HASH' for the same indicators. For consistency, please ensure the types in the human-readable output match the types in the context output.